_gsocproposals/proposal_GoHEPInterpreter.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Launching Wagon, a WebAssembly interpreter in Go
+layout: plain
+project: Go-HEP
+organization: CERN
```

Comment:
It is a fastly moving target!!! As soon as #90 is merged, it should be very easy to add an organization but the organization should be a real one (a legal entity) I guess. Adding @hegner who knows probably better the constraints.